v4/core/log.go

```diff
@@ -36,6 +36,8 @@ const (
 // Users of the library can supply their own implementation by calling SetLogger().
 type Logger interface {
 	Log(level LogLevel, format string, inserts ...interface{})
+	// TODO: add in next major release
```

Comment:
In @dpopp07 's go core PR  re: logging, I recommended some other methods be added to the Logger interface as well, but completely whiffed on the fact that it would introduce a breaking change.
I think we'll need to add methods to the Logger interface in order to solve Dustin's issue the "right way".
We could avoid creating a major version and take a chance that nobody has implemented their own Logger impl (probably not preferable), but it wouldn't be all that strenuous to just create a new major version (v5) of the go core (and introduce SetLogLevel() and any changes from Dustin's PR, then update the generator (as part of Dustin's logging-related generator PR) to generate Go code that references go core v5 instead of v4.